pkg/reconciler/openshift/tektonconfig/rbac.go

```diff
@@ -393,54 +396,73 @@ func (r *rbac) createResources(ctx context.Context) error {
 		return err
 	}
 
+	var wg sync.WaitGroup
+	sem := make(chan struct{}, rbacMaxConcurrentCalls) // Semaphore with buffer size of rbacMaxConcurrentCalls
```

Comment:
Why do we need this if we use `sync.WaitGroup` ?

Reply:
@vdemeester I am using semaphore to set a maximum number of concurrent goroutine to avoid overloading the api server as processResourcesForSingleNamespace is doing a couple of calls to apiserver